src/test/java/core/basesyntax/model/FruitTest.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.model;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+import core.basesyntax.ValidationException;
+import org.junit.Test;
+
+public class FruitTest {
```

Comment:
I dont think we need to test Fruit model if we don't have some not standard logic there

Reply:
Can i keep it though? :)

Reply:
lets exclude model classes from test coverage and remove this test class